TypeCobol/Compiler/FileCompiler.cs

```diff
@@ -79,6 +80,33 @@ public class FileCompiler
 
         public EventHandler<ExecutionStepEventArgs> ExecutionStepEventHandler { get; set; }
 
+        private ILanguageServer _LanguageServer;
+        /// <summary>
+        /// Any connection to a Language Server instance.
+        /// </summary>
+        public ILanguageServer LanguageServer
```

Comment:
A property "LanguageServer" shouldn't be in TypeCobol project.
Can you move it to TypeCobol.LanguageServer project ?

Reply:
It's done,  with code refactoring.